src/tribler-core/tribler_core/components/restapi/rest/debug_endpoint.py

```diff
@@ -41,12 +44,12 @@ class DebugEndpoint(RESTEndpoint):
     This endpoint is responsible for handing requests regarding debug information in Tribler.
     """
 
-    def __init__(self, *args, **kwargs):
-        RESTEndpoint.__init__(self, *args, **kwargs)
-        self.state_dir = None
-        self.log_dir = None
-        self.tunnel_community = None
-        self.resource_monitor = None
+    def __init__(self, state_dir: Path, log_dir: Path, tunnel_community:TunnelCommunity=None, resource_monitor:ResourceMonitor=None):
```

Comment:
**Suggestion:** I'm not sure why pylint is not raising an issue here but we usually have a space after the `:` when using `typing`. For consistency, consider changing this for the last two attributes.

Reply:
Again, the linter was silent... :sheep: